### DISCUSSION TITLE
Add social media post in article view

### DISCUSSION THREAD

**Discussion by Tunsinasro** [spacecowboy/Feeder#159] [Ideas]
I think it would be nice to view social media posts (or a snipet of the post) in article view instead if showing a url, the same as feedme app

**Comment by spacecowboy** [comment]
I don't really understand what exactly you are referring to.

But the words "social media posts" makes me immediately want to say: absolutely not

**Comment by Tunsinasro** [comment]
When an article contains a tweet for example, it would be nice to show it instead of a link to the tweet, look at the screenshot bellow
![Screenshot_2023-12-21-14-22-07-928_com seazon feedme](https://github.com/spacecowboy/Feeder/assets/147873384/5010a1fe-7a9c-4e3b-adc4-a68577b5f6a4)

**Reply by spacecowboy** [reply]
Can't be done. That would require Javascript execution

**Reply by Tunsinasro** [reply]
Ok got it, thank you nonetheless
